models/models.js

```diff
@@ -76,3 +76,26 @@ exports.fetchCommentsByArticle = (id) => {
         });
     });
 };
+
+exports.createComment = ({ commentBody, username }, id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [id])
+    .then(({ rows }) => {
```

Comment:
I can understand why you're doing this and it would be a nice way to check if article exists, however....
Both `article_id` and `author` are _foreign keys_ that reference other tables in the database, which means you only need the one query to `INSERT INTO comments...` and if they don't exist PSQL will throw the error for you